# Bioinformatics-resources
in here you will find a huge number of useful Bioinformatics &amp; computational biology tools  

-VLS3D.com      website full with resources (contain more than 1000 tools) 

-RNA 22    (hybridisation) 

-psRNATarget (hybridisation) 

-Tapir target (hybridisation) 

-Miranda (hybridisation) 

-RNAHyprid (hybridisation) 

-Target scan (hybridisation) 

-I tasser  (protein structure prediction )

-Psipred (protein secondary structure prediction )

-Alpha fold  (protein structure prediction)

-Swiss model (protein structure prediction and many other tools )

-Expasy (portal contain many useful tools)

-RNA fold 

-Rosetta (prediction)

-Robetta (prediction)

-Venny diagram (to make scientific diagrams )

-Chord diagram  by R  (scientific diagram)

-Mirbase (to get micro RNA, contain multiple organisms, for hybridisation purpose)

-NCBI virus (to retrieve viruses data ) 

-NCBI  (to retrieve biological data)	

-BLAST (it is a known tool ) 

-Pymol (to show structure of molecules )

-RasMol

-USCF Chimera

-VMD

-Pymol to visually analyse a pdb (intereaction between ligand and protein, hydrogen and salt bonds, secondary structures etc...). Chimera to work with my virus capsids and produce videos, and VMD to work with molecular dynamics

-Therapeutic target database (Database for drug discovery purpose) 

-Plantcare database

-EMBL 
AFLPinSilico performs AFLP simulations with different combinations of restriction enzymes or selective nucleotides, on organisms or data sets for which DNA sequences are available.

-ConCysFind is searching conserved amino acids in proteins of plants.

-ClustalW - general purpose multiple sequence alignment program for DNA or proteins

-acdc - (A)utomated (c)ontamination (d)etection and (c)onfidence estimation for single-cell genome data. Acdc is a tool to test next-generation-sequencing (NGS) data from single-cell sequencing for contamination. By using sophisticated dimensionality reduction and clustering methods, it uses tetramer profiles to differentiate between different species in a given sample. It automatically detects the number of clusters/species and provides confidence information.

-jPREdictor - jPREdictor is a tool for the prediction of cis-regulatory elements, e.g. PRE/TREs in Drosophila melanogaster

-PCR Primer Design

-genefisher2 - A Tool for Backtranslation and ambiguous Consensus calculation

-RNAforester - RNAforester is a tool for RNA secondary structure comparison

-CB-DOCK (molecular docking)

-Dock thor (virtual screening )

-PRANKweb based from p2rank (predict binding sites )

-Robetta (protein structure prediction) 

-ZINC database (ligands database)

-Chembl database (ligands database)

-DRUG bank (ligands database)

-AUTO-Dock vina (virtual screening tool)

-BLAST 
It is a search tool, used for DNA or protein sequence search based on identity. 

-HMMER Homologous protein sequences may be searched from the respective databases using this tool. 

-Clustal Omega 
Multiple sequence alignments may be performed using this program

-Sequerome 
Used for sequence profiling. 

-ProtParam 
Used to predict the physico-chemical properties of proteins. 

-JIGSAW 
To find genes, and to predict the splicing sites in the selected DNA sequences.

-novoSNP 
Used to find the single nucleotide variation in the DNA sequence. 

-ORF Finder 
The putative genes may be subjected to this tool to find Open Reading Frame (ORF).http://www.ncbi.nlm. nih.gov/projects/gorf/

-PPP 
Prokaryotic promoter prediction tool used to predict the promoter sequences present up-stream the gene http://bioinformatics. biol.rug.nl/websoftware/ppp/ppp_start.php

-Virtual Foorprint 
Whole prokaryotic genome (with one regular pattern) may be analysed using this program along with promoter regions with several regulator patterns. 

-WebGeSTer 
This is a database containing sequences of transcription terminator sequences and is used to predict the termination sites of the genes during transcription. 

-Genscan 
Used to predict the exon-intron sites in genomic sequences. 

-Softberry Tools 
Several tools are specialized in annotation of animal, plant, and bacterial genomes along with the structure and function prediction of RNA and proteins.

-MEGA (Molecular Evolutionary Genetics Analysis) 
Builds phylogenetic tress to study the evolutionary closeness. 

-MOLPHY 
It is molecular phylogenetic analysis tool based on maximum likelihood method. 

-PAML 
A phylogenetic analysis tool based on maximum likelihood. 

-PHYLIP 
A package for phylogenetic studies. 

-JStree 
An open-source library for viewing and editing phylogenetic trees for presentation improvement. 

-TreeView 
Software to view the phylogenetic trees, with the provision of changing view. 

-Jalview 
It is an alignment editor and is used to refine the alignment 


- Nucleotide Databases 

DNA Data Bank of Japan 
It is the member of International Nucleotide Sequence Databases (INSD) and is one of the biggest resources for nucleotide sequences. 
 
European Nucleotide 
Archive It captures and presents information relating to experimental workflows that are based around nucleotide sequencing. 

GenBank 
It is the member of International Nucleotide Sequence Databases (INSD) and is a nucleotide sequence resource. 

Rfam 
A collection of RNA families, represented by multiple sequence alignments

- Protein Databases 

Uniprot One of the largest collection of protein sequences. 

Protein Data Bank 
This is another major resource of proteins containing information of experimentally-determined structures of nucleic acids, proteins, and other complex assemblies. 

Prosite 
Provides information on protein families, conserved domains and actives sites of the proteins. 

Pfam 
Collection of protein families 

SWISS PROT 
A section of the UniProt Knowledgebase containing the manually annotated protein sequences 
 
InterPro 
Describes the protein families, conserved domains and actives sites 

Proteomics Identifications 
Database A public source, containing supporting evidence for functional characterization and post-translation modification of proteins and peptides.

- Genome databases 

Ensembl 
It is a database containing annotated genomes of eukaryotes including human, mouse and other vertebrates. 

PIR 
An integrated public resource to support genomic and proteomic research

- Miscellaneous Databases 

Medherb 
Resource database for medicinally important herbs 

Reactome 
A peer-reviewed resource of human biological processes 
 
TextPresso 
This database provides full text literature searches of model organism research, helps database curators to identify and extract biological entities which include new allele and gene names and human disease gene orthologs http://www.textpresso.org
 

TAIR 
The Arabidopsis Information Resource (TAIR) maintains a database of genetic and molecular data for the model plant Arabidopsis thaliana. It provides information on gene structure, gene product, gene expression, DNA and seed stocks, genome maps, genetic and physical markers. http://www. arabidopsis.org

dictyBase 
dictyBase is an online bioinformatics database for Dictyostelium discoideum. 

- Signalling & Metabolic Pathway Databases 

KEGG 
KEGG is a suite of databases and associated software for understanding and simulating higher-order functional behaviours of the cell or the organism from its genome information. 

CMAP 
Complement Map Database is a novel and easily accessible research tool to assist the complement community and scientists from related disciplines in exploring the complement network and discovering new connections. 

SGMP 
The Signaling Gateway Molecule Pages (SGMP) database provides highly structured data on proteins which exist in different functional states participating in signal transduction pathways. 

PID 
The Pathway Interaction Database (PID) is a collection of curated and peer-reviewed pathways composed of human molecular signaling and regulatory events and key cellular processes. It serves as a research to study the cellular pathways with a special emphasis on cancer.

HMDB 
The Human Metabolome Database (HMDB) is the most comprehensive curated collection of human metabolite and human metabolism data in the world. It contains records for more than 2180 endogenous metabolites with information gathered from thousands of books, journal articles and electronic databases along an extensive collection of experimental metabolite concentration data compiled from hundreds of mass spectra (MS) and Nuclear Magnetic resonance (NMR) from the analyses performed on urine, blood and cerebrospinal fluid samples. The HMDB is designed to address the broad needs of biochemists, clinical chemists, physicians, medical geneticists, nutritionists and members of the metabolomics community



-CATH 
A semi-automatic tool for the categorized organization of proteins. 

-RaptorX 
It facilitates the user to predict protein structure based on either a single- or multi-template threading. 

-JPRED 
Used to predict secondary structures of proteins. 

-PHD 
Used to predict neural network structure.

-HMMSTR 
A hidden Markov model for the prediction of sequence-structure correlations in proteins. 

-APSSP2 
Predicts the secondary structure of proteins. 

-MODELLER 
Predicts 3D structure of protein based on comparative modelling 

-Phyre and Phyre2 
Web-based servers for protein structure prediction

-SMART 
A Simple Modular Architecture Retrieval Tool; describes multiple information about the protein query. 

-AutoDock 
Predicts protein-ligad interaction and is considered as reliable tool. 

-HADDOCK 
Describes the modelling and interaction of bio-molecular complexes such as protein-protein, protein-DNA 

-BIND 
A database that provides access to molecular interaction and bio-complexes 

-MOE 
An integrated package of tools used for drug discovery. It combines visualization, modelling, and drug discovery on one plate-form. 

-STRING 
A database of both known and predicted protein interactions. 

-MIMO 
A dynamics graph-matching tool for the comparison of biological pathways in an efficient manner. 

-IntAct 
It is an open source database system and provides analysis tools for molecular interaction data. All interactions are derived from literature curation or direct user submissions and are freely available. 

-Graemlin 
It is capable of scalable multiple network alignment with its functional evolution model that allows both the generalization of existing alignment scoring schemes and the location of conserved network topologies other than protein complexes and metabolic pathways. 

-PathBLAST It is meant to search protein-protein interaction network of the any selected organism and extracts all interaction pathways that align with the query. 

- CFinder 
This tool is capable of finding and visualizing the overlapping dense groups of nodes in networks, and quantitative description of the evolution of social groups. It is efficient for clustering data represented by genetic or social networks and microarray. 

-MCODE 
It is suited for both computationally and biologically oriented researchers. Its features include; Fast network clustering, Fine-tuning of results with numerous node-scoring and cluster-finding parameters, Interactive cluster boundary and content exploration, Multiple result set management, Cluster sub-network creation and plain text export Potential Drug Target 

-Database (PDTD) 
It is a dual function, wide-range database of drug targets that is globally accessible via internet, containing 1207 entries including 842 with known structures.

-Drug Bank 
It is an exclusive resource that interconnects the drug-target data. It contains 7681 drug entries including 1545 FDA-approved, 155 FDA-approved biotech-based drugs, 89 nutraceuticals and over 6000 experimental drugs along with 4218 non-redundant protein sequences which are associated to these drug entries.

-Therapeutic Target 
Database (TTD) It is a collection of known and explored therapeutic proteins and DNA targets, the disease, pathways involved in the disease and the corresponding drugs directed at each of these targets. It also provides links to relevant databases about target function. 

-TDR Target Database 
It is a database as well as a tool. It is meant to identify and prioritize the genes of interest from the ignored disease pathogens by running simple queries, assigning them numerical values and combining the output to produce a ranked list of candidate targets. The TDR here is abbreviated for Tropical Disease Research which is a special program within the World Health Organization (WHO) agenda. 

-MATADOR: Manually Annotated Targets and Drugs Online Resource MATADOR is a resource for protein-chemical interactions. In contrast to DrugBank which usually contains only the main mode of interaction, the MATADOR provides manually annotated list of direct (binding) as well as protein-chemical interaction. Each interaction is linked to PubMed or OMIM entries that were used to deduce the interaction. The user can choose either to trust only the direct interactions (with a known mechanism) or also indirect interactions. 

-TB Drug Target Database 
This is a specialized database contains information on drugs and target proteins for the treatment of tuberculosis (TB) only including the structural details of inhibitors. http://www.bioinformatics.org/ tbdtdb/

-DrugPort 
It provides the structural information available in the Protein Data Bank (PDB) related to drug molecules and their targets based on the latest version of DrugBank database. 
http://www.ebi.ac.uk/thorntonsrv/databases/drugport/

-ChEMBL 
It is a collection of drug-like bioactive molecules, along with their 2-D structures, calculated and abstracted properties such as; logP, molecular mass, Lipinski Parameters, binding constants, pharmacokinetics etc.

-Abalone 
Abalone is a general purpose molecular modeling program which is meant for biomolecular dynamics simulations of proteins, DNA, ligands. It has by-default ability to interact with external quantum programs NWChem, CP2K and PC GAMESS/Firefly). http://www.biomolecular-modeling. com/Abalone/index.html 

-Ascalaph 
Similar to Abalone, Ascalaph is also a general purpose molecular modeling tool to perform quantum mechanics calculations for model development, molecular mechanics and dynamics simulations of DNA, proteins and hydrocarbons, either in the gas or in condensed phase. It has a built-in ability to interact with external molecular modeling packages such as, (MDynaMix, ORCA, NWChem, CP2K, PC GAMESS/ Firefly and DelPhi). http://www.biomolecular-modeling. com/Products.html 

-Discovery Studio 
Discovery Studio is a comprehensive modeling and simulation package focused on optimizing the drug discovery process including the capabilities of small molecule simulations, pharmacophore modelling, protein-ligand docking, protein homology modelling, sequence analyses, protein-protein docking and antibody modelling, etc. http://accelrys.com/products/ discovery-studio/

-Amber 
Amber is the collection of programs that facilitate users to perform molecular dynamics simulations with an emphasis on biomolecules. 

-FoldX 
FoldX provides quick and quantitative estimation of molecular interactions which are contributing towards the stability of either single protein or protein complexes.





